README.md

```diff
@@ -64,6 +64,16 @@ The easiest way to interact with the Plug and Play sample from the IoT service s
 
 Additional instructions for using Azure IoT Explorer are available [here](https://github.com/Azure/azure-iot-explorer/#plug-and-play).
 
+### Azure Device Update
+
+You can interact with the Azure Device Update, remotely updating your Azure IoT Hub devices through the samples below:
+
+- ESPRESSIF:
+  - [ESP32](demos/projects/ESPRESSIF/adu/README.md)
+
+- PC Simulation:
+  - [Linux](demos/projects/PC/linux/ADU.md)
```

Comment:
like @vaavva mentioned should probably have the L475 as well

Reply:
Added.